client/quantum_serverless/core/job.py

```diff
@@ -131,21 +132,13 @@ def list(self, **kwargs) -> List["Job"]:
 
     def run(self, program: Program, arguments: Optional[Dict[str, Any]] = None):
         arguments = arguments or {}
-        arguments_string = ""
-        if arguments is not None:
-            arg_list = []
-            for key, value in arguments.items():
-                if isinstance(value, dict):
-                    arg_list.append(f"--{key}='{json.dumps(value)}'")
-                else:
-                    arg_list.append(f"--{key}={value}")
-            arguments_string = " ".join(arg_list)
-        entrypoint = f"python {program.entrypoint} {arguments_string}"
+        entrypoint = f"python {program.entrypoint}"
 
         # set program name so OT can use it as parent span name
```

Comment:
nit: update comment as we're not just updating program name anymore (?)

Reply:
what do you mean? :)

Reply:
github UI :anger: 

that referred to the comment on L137: it only refers to setting the program name, but now we're also setting job arguments...

Reply:
ah! :) now I understand. You are right, comment needs to be updated/removed. Will update it in any of next PRs :)